components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineView.kt

```diff
@@ -119,6 +124,31 @@ interface EngineView {
      * A delegate that will handle interactions with text selection context menus.
      */
     var selectionActionDelegate: SelectionActionDelegate?
+
+    /**
+     * Enumeration of all possible ways user's [android.view.MotionEvent] was handled by GeckoView.
```

Comment:
The concept components do not reference a specific implementation. Maybe we can make this comment more generic?

Same for the `@see` references.

Reply:
Thanks!